src/mod/applications/mod_voicemail/mod_voicemail.c

```diff
@@ -3402,11 +3402,13 @@ static switch_status_t voicemail_leave_main(switch_core_session_t *session, vm_p
 	switch_bool_t skip_greeting = switch_true(switch_channel_get_variable(channel, "skip_greeting"));
 	switch_bool_t skip_instructions = switch_true(switch_channel_get_variable(channel, "skip_instructions"));
 	switch_bool_t skip_record_urgent_check = switch_true(switch_channel_get_variable(channel, "skip_record_urgent_check"));
+	switch_bool_t voicemail_skip_goodbye = switch_true(switch_channel_get_variable(channel, "voicemail_skip_goodbye"));
```

Comment:
Please use `switch_channel_var_true()` instead.

Reply:
sure, also the other variables?

Reply:
makes sense

Reply:
done for all.